requirements.txt

```diff
@@ -60,5 +60,7 @@ bs4==0.0.1
 html==1.16
 checksumdir==1.1.0
 
+elasticsearch==5.4.0
+
```

Comment:
Now, if we are using separate container for **elasticsearch**, please `checkout` this file.